include/ignition/math/Vector2.hh

```diff
@@ -134,6 +136,42 @@ namespace ignition
         return (this->data[0] * _v[0]) + (this->data[1] * _v[1]);
       }
 
+      /// \brief Set this vector's components to the maximum of itself and the
+      ///        passed in vector
+      /// \return[in] _v the maximum clamping vector
+      public: void Max(const Vector2<T> &_v)
+      {
+        if (_v[0] > this->data[0])
+          this->data[0] = _v[0];
+        if (_v[1] > this->data[1])
+          this->data[1] = _v[1];
+      }
+
+      /// \brief Set this vector's components to the minimum of itself and the
+      ///        passed in vector
+      /// \return[in] _v the minimum clamping vector
```

Comment:
This function doesn't return, this should probably be `\param`

Reply:
done!